content/Projects/python-prometheus-api-client.md

```diff
@@ -0,0 +1,15 @@
+# python-prometheus-api-client
```

Comment:
nit: the project is listed as `python-prometheus-api-client` here but `prometheus-api-client-python` further down. Can we pick one consistent name? Ideally the name of the project on its actual page so that users are not confused when they click through the link